op-service/txmgr/send_state.go

```diff
@@ -57,11 +59,14 @@ func (s *SendState) ProcessSendError(err error) {
 	defer s.mu.Unlock()
 
 	// Record the type of error
+	s.alreadyReserved = false
```

Comment:
Is the idea here to reset it to false in case there's an error different from `ErrAlreadyReserved`, which would imply that it's something with nonces, low fees etc?

Not sure if this works well, because also RPC errors etc would land here and would then reset it.

Reply:
Removed. This is leftover for when I had it waiting for the mempool to clear (you only care about the most recent attempt), but since the abort check in the retry loop is now immediate it wouldn't have mattered.